### PR TITLE
Kunal/fix scrollbar

### LINKED ISSUE #2171: Redundant Scrollbar
Currently when you open the setting tab you see a scrollbar even in the case when there is nothing to scroll to. This isn't a great user experience. The fix should be that the scrollbar should only appear when required.
image

### PR DESCRIPTION
**Pull request by kunxl-gg**
Fixes #2171 

Changes: I have changed the css to show the scrollbar only when necessary. 
image
image

I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] has no test errors (`npm run test`)
* [x] is from a uniquely-named feature branch and is up to date with the  `develop` branch.
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

### CODE REVIEW

File: client/styles/components/_preferences.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
     flex-direction: column;
   }
   & .react-tabs__tab-panel {
-    overflow-y: scroll;
+    overflow-y: auto;
```
Comment (lindapaiste) [inline client/styles/components/_preferences.scss:20]: Perfect! 👍

File: index.js [hunk 2]
```diff
@@ -18,3 +18,4 @@ if (process.env.NODE_ENV === 'production') {
   require('./server/server');
   require('./server/previewServer');
 }
+
```
Comment (lindapaiste) [inline index.js]: If possible, can you revert this file?

**Review by lindapaiste — DISMISSED** [review]

File: index.js (hunk 2 above)
Comment (kunxl-gg) [reply index.js]: @lindapaiste I think I added an extra line by accident. Fixed that.

**Comment by kunxl-gg** [comment]
@lindapaiste is there anything else I need to fix in the PR.

**Comment by lindapaiste** [comment]
> @lindapaiste is there anything else I need to fix in the PR.

Nope, it looks great to me!  The next step it that it needs to be approved by someone who is a maintainer - probably @raclim

**Review by raclim — APPROVED** [review]

**Pull request merged**